fedimint-testing/src/ln/mock.rs

```diff
@@ -163,4 +163,28 @@ impl ILnRpcClient for FakeLightningTest {
     ) -> Result<EmptyResponse, LightningRpcError> {
         Ok(EmptyResponse {})
     }
+
+    async fn create_invoice(
+        &self,
+        create_invoice_request: CreateInvoiceRequest,
+    ) -> Result<CreateInvoiceResponse, LightningRpcError> {
+        let ctx = bitcoin::secp256k1::Secp256k1::new();
+
+        let payment_hash = sha256::Hash::from_slice(&create_invoice_request.payment_hash)
+            .expect("Failed to lookup FederationId");
+        let invoice = InvoiceBuilder::new(Currency::Regtest)
+            .description("".to_string())
+            .payment_hash(payment_hash)
+            .current_timestamp()
+            .min_final_cltv_expiry_delta(0)
+            .payment_secret(PaymentSecret([0; 32]))
```

Comment:
Note: going forward, if we want to support MPP payments I think we'll need to set this correctly. Obviously not blocking for now.